### PR TITLE
Various Edits for content management purposes

### PR DESCRIPTION
**Pull request by TylerTrott**
**- What I did**
- Fixed up a few links, titles, and files
- Completed the SnackBar demo app file content
- Added more files for demo apps

**- How I did it**
Local edits

**- How to verify it**
Verify on devstaging

**- Description for the changelog**
Various Edits for content management purposes

### CODE REVIEW

File: content/en/docs/Get-Started/TryAtPlatform.md [hunk 1]
```diff
@@ -139,6 +139,11 @@ Watch the GIF below to see how to get a free @sign within the app itself! If you
   <!-- End of Steps column -->
 
 </div>
+<br>
+
+### What next?
```
Comment (xavierchanth) [inline content/en/docs/Get-Started/TryAtPlatform.md:144]: typo? is it "What's next?"

File: content/en/docs/Sample_Apps/atmosphere.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Atmosphere Pro"
```
Comment (xavierchanth) [inline content/en/docs/Sample_Apps/atmosphere.md:2]: Should the title match and be "@‎mosphere pro"?

**Review by xavierchanth — CHANGES_REQUESTED** [review]
LGTM, but I did ask a few questions left for input.

**Review by anthonyvprakash — APPROVED** [review]

File: content/en/docs/Sample_Apps/atmosphere.md (hunk 2 above)
Comment (anthonyvprakash) [reply content/en/docs/Sample_Apps/atmosphere.md:2]: Yep we should

File: content/en/docs/Get-Started/TryAtPlatform.md (hunk 1 above)
Comment (TylerTrott) [reply content/en/docs/Get-Started/TryAtPlatform.md:144]: will re-raise with edits

Comment (xavierchanth) [reply content/en/docs/Get-Started/TryAtPlatform.md:144]: @TylerTrott in the future, you can leave the PR open and add more commits to the same branch, they will be added to the PR.

**Pull request closed without merging**